scripts/regression_test_python.py

```diff
@@ -259,9 +291,19 @@ def test_arrow_dictionaries_scan():
             write_result(benchmark_name, run_number, duration)
 
 
+def test_loading_pandas_df_many_times():
+    test = PandasDFLoadBenchmark()
+    results = test.benchmark()
+    benchmark_name = f"load pandas df many times"
```

Comment:
I think we use duckdb to parse the benchmarks, I don't know if the script deals with spaces in the benchmark name, probably best to replace them with underscores